cmd/heartbeats/main.go

```diff
@@ -70,7 +70,7 @@ func main() {
 func cloudEventsContext() *cloudevents.EventContext {
 	return &cloudevents.EventContext{
 		CloudEventsVersion: cloudevents.CloudEventsVersion,
-		EventType:          "dev.knative.source.heartbeats",
+		EventType:          "dev.knative.eventing.heartbeats",
```

Comment:
Apologies if the rest of community is aware, but since I am new, I will ask. Why `dev` is used in the prefix? Is it indication of wip, or not stable api which will be change close to v1.0, or something else?

Reply:
Someone may correct me, but I believe https://knative.dev is just the domain we registered for knative, so that's used for namespacing things. Dev here just meaning it's a developer tool/thing, not an indication of maturity.